web_hosting_manager/app/components/app_logs.js

```diff
@@ -13,28 +12,30 @@ export default class AppLogs extends Component {
       error: ''
     };
   }
-  componentDidMount() {
-    this.setState({loading: true});
+
+  componentWillMount() {
+    this.setState({ loading: true });
     this.props.getLogFilePath()
       .then(path => {
         this.logPath = path.value;
         this.readLogFile();
-        this.setState({loading: false});
+        this.setState({ loading: false });
+        return Promise.resolve(true);
```

Comment:
I think we've moved to use async/await so we should keep consistency here as well.

Reply:
I've updated the code gabriel..